frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -830,6 +837,9 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
             },
         ],
     }),
+    afterMount(({ cache }) => {
```

Comment:
Possibly some browsers with stale `cache` values that are being populated on load (e.g. values not removed in the `beforeUnmount` somehow). Reset all timing metrics on mount

Reply:
🧠